scripts/requirements_override.py

```diff
@@ -0,0 +1,46 @@
+"""Script for copying python version-specific requirements files to override requirements.txt.
+
+This is for pipelines only and should not be run by developers on their machines.
```

Comment:
I believe that this looks for a `requirements-{m}.{n}.txt` file and matches m & n to the version of Python running this script? Again, this is another place where an extra comment would help (or better.... `desc` in the `parser` :-)

Reply:
I'll tackle that in a follow-up since I don't want to keep this one lingering for longer.